niri-config/src/output.rs

```diff
@@ -248,6 +248,7 @@ impl<S: ErrorSpan> Decode<S> for Modeline {
             hsync_end,
             htotal,
         );
+        ensure!(0u16 < htotal, ctx, htotal_value, "htotal {} > 0", htotal);
```

Comment:
Consistency with other messages please. `htotal {} must be > 0` Here and in other places